float32 Multi
uint8 calibGrip
uint8 sys_flag
uint8 calibID
uint8 calibArm
